packages/api/endpoints/granules.js

```diff
@@ -684,47 +672,16 @@ const associateExecution = async (req, res) => {
 async function delByGranuleId(req, res) {
   const {
     knex = await getKnexClient(),
-    esClient = await getEsClient(),
-    esGranulesClient = new Search({}, 'granule', process.env.ES_INDEX),
   } = req.testContext || {};
 
   const granuleId = req.params.granuleId;
   log.info(`granules.del ${granuleId}`);
 
-  let pgGranule;
-  let esResult;
-  try {
-    // TODO - Phase 3 - we need a ticket to address granule/collection consistency
-    // For now use granule ID without collection search in ES
-    pgGranule = await getUniqueGranuleByGranuleId(knex, granuleId);
-  } catch (error) {
-    if (error instanceof RecordDoesNotExist) {
-      // TODO - Phase 3 - we need to require the collectionID, not infer it
-
-      esResult = await esGranulesClient.get(granuleId);
-
-      if (esResult.detail === recordNotFoundString) {
-        log.info('Granule does not exist in Elasticsearch and PostgreSQL');
-        return res.boom.notFound('No record found');
-      }
-      if (esResult.detail === multipleRecordFoundString) {
-        return res.boom.notFound(
-          'No Postgres record found, multiple ES entries found for deletion'
-        );
-      }
-      log.info(
-        `Postgres Granule with ID ${granuleId} does not exist but exists in Elasticsearch.  Proceeding to remove from elasticsearch.`
-      );
-    } else {
-      throw error;
-    }
-  }
+  const pgGranule = await getUniqueGranuleByGranuleId(knex, granuleId);
```

Comment:
We should keep the try catch for pgGranule, and be able to return
res.boom.notFound('No record found');

Same for function del